internal/sudo/contract_manager.go

```diff
@@ -75,37 +74,72 @@ type OpenAckDetails struct {
 	CounterpartyVersion   string `json:"counterparty_version"`
 }
 
+type ContractMethods interface {
+	HasContractInfo(sdk.Context, sdk.AccAddress) bool
+	Sudo(sdk.Context, sdk.AccAddress, []byte) ([]byte, error)
+}
+
+var _ ContractMethods = (*ContractManager)(nil)
+
+type ContractManager struct {
+	wasmKeeper ContractMethods
```

Comment:
what do you think about renaming the field so it looks more like the interface name? wasmKeeper imo doesn't fit here

Reply:
agree